installer/manifests/keptn/charts/control-plane/templates/rbac.yaml

```diff
@@ -212,6 +212,29 @@ rules:
       - delete
       - deletecollection
 
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: keptn-acquire-lease
+  labels:
+    app.kubernetes.io/name: keptn-acquire-lease
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/part-of: keptn-{{ .Release.Namespace }}
+    app.kubernetes.io/component: {{ include "control-plane.name" . }}
+    helm.sh/chart: {{ include "control-plane.chart" . }}
+rules:
+  - apiGroups:
+      - "coordination.k8s.io"
+    resources:
```

Comment:
here we can limit the access to the specific resource name (shipyard-controller-dispatcher)